.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+- id: cpplint
+  name: cpplint
+  description: Check style of C code using cpplint.
+  language: python
+  types_or: [c++, c]
```

Comment:
Would you mind adding cuda files as well?
`types_or: [c++, c, cuda]`
We use cpplint to check our C++ and CUDA files. It would be nice to have it on by default, otherwise it will be required to change the hook configuration